EstateAdministrationUI/Startup.cs

```diff
@@ -30,6 +42,7 @@
                                                                       .AddJsonFile($"appsettings.{webHostEnvironment.EnvironmentName}.json", optional: true).AddEnvironmentVariables();
 
             Startup.Configuration = builder.Build();
+            Startup.WebHostEnvironment = webHostEnvironment;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this assignment of 'WebHostEnvironment' or initialize it statically.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=5485852)